runtime/test/package.go

```diff
@@ -0,0 +1,13 @@
+// Package test provides a number of functions and tools for writing fully integrated test suites for Encore
+// applications.
+package test
```

Comment:
I think test is fine personally. 

It doesn’t overlap with go’s testing package and if it conflicts with a customers test helpers package; then it can be aliased. 

When we make the expectation and matcher api’s would we call the packages ematching & eexpect. 

I think that impacts readability too much. Given go allows for aliasing of imports I’m not overly concerned by this